src/OnboardingSPA/components/HeadingWithSubHeading/index.js

```diff
@@ -3,20 +3,25 @@ import { __ } from '@wordpress/i18n';
 /**
  * Interface Cards with standard design.
  *
- * @returns
+ * @param {Object} root0
+ * @param {string} root0.title
+ * @param {string} root0.subtitle
+ * @param {Object} root0.children
  */
-const HeadingWithSubHeading = ({ title, subtitle }) => {
-
+const HeadingWithSubHeading = ( { title, subtitle, children } ) => {
 	return (
 		<div className="nfd-main-heading">
-			<h2 className="nfd-main-heading__title">{__(
-				title,
-				"wp-module-onboarding"
-			)}</h2>
-			<h3 className="nfd-main-heading__subtitle">{__(
-				subtitle,
-				"wp-module-onboarding"
-			)}</h3>
+			<h2 className="nfd-main-heading__title">
+				{ /* eslint-disable-next-line @wordpress/i18n-no-variables */ }
+				{ __( title, 'wp-module-onboarding' ) }
+			</h2>
+			{ subtitle && (
+				<h3 className="nfd-main-heading__subtitle">
+					{ /* eslint-disable-next-line @wordpress/i18n-no-variables */ }
+					{ __( subtitle, 'wp-module-onboarding' ) }
```

Comment:
We should remove the translation here and fix this one instead of disabling the lint check.

Reply:
Was using this as a fallback but as the translations story is in, I'll change that.

Reply:
Done